### PR TITLE
[bulk] Validations: fixes duplicate-H1-Title issues

### PR DESCRIPTION
**Pull request by Vette99**
This PR fixes 10 duplicate -H1-Title issues.

### CODE REVIEW

File: dynamics-nav-app/LocalFunctionality/Belgium/how-to-set-up-declaration-types.md [hunk 1]
```diff
@@ -23,16 +23,16 @@ In [!INCLUDE[navnow](../../includes/navnow_md.md)], there are two types of decla
...
-When using the extended declaration, you will also need to set up an Incoterm in Intrastat Declaration for each shipping method. For more information, see [Incoterm in Intrastat Decl. Field](ms-its:be_lf_a.chm::/T_10_11300.htm).  
+When using the extended declaration, you will also need to set up an Incoterm in Intrastat Declaration for each shipping method.
 
 ## To set up declaration types  
 
 1.  Choose the ![Search for Page or Report](../../media/ui-search/search_small.png "Search for Page or Report icon") icon, enter **General Ledger Setup**, and then choose the related link.  
 2.  On the **General** FastTab, select the **Simplified Intrastat Decl.** check box to set up a simplified declaration type. Clear this field to use extended declaration.  
 3.  Choose the **OK** button.  
-
+microsoft.com
```
Comment (dereklh7) [inline dynamics-nav-app/LocalFunctionality/Belgium/how-to-set-up-declaration-types.md:33]: Why was this added? Could be removed.

Comment (Vette99) [reply dynamics-nav-app/LocalFunctionality/Belgium/how-to-set-up-declaration-types.md]: Don't know how that got there, but I took it out as suggested. Thanks.

**Comment by Vette99** [comment]
@Eva Dupont
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Review by edupont04 — APPROVED** [review]
